src/Maestro/tests/scenarios.md

```diff
@@ -0,0 +1,131 @@
+# Maestro++ Scenarios and test coverage
+
+## Overview
+
+This is a list of the business scenarios that Maestro is involved in, and the automated or manual validation to be performed for those scenarios
+
+## Package Publishing
+
+Maestro is only involved in the insertion of builds into the BAR.
```

Comment:
Does this mean that there's no scenario to test?  Not sure how to interpret the statement

Reply:
Yeah, basically struggled with conveying a conclusion about every section. IN this case, I meant that we only care about build creation, and that is tested both by the scenario test that I mention, and the post requests to the API. I'll reword